storage_service/locations/models/wellcome.py

```diff
@@ -133,22 +180,32 @@ def move_from_storage_service(self, src_path, dest_path, package=None):
             location = package.current_location
             space_id = location.relative_path.strip(os.path.sep)
 
+            # For reingests, the package status will still be 'uploaded'
+            # We want to use this to detect when upload is complete,
```

Comment:
"We want to use this" – is "this" the package status? This comment took me a moment to understand.